pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```

Comment:
Is it downgrade?  🤔

Reply:
pnpm v8.6.2 reverted the lockfile update seemingly: https://github.com/pnpm/pnpm/releases/tag/v8.6.2
interesting decision for sure, I was confused by that right now as well.